core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -904,7 +904,7 @@ wasm_interp_call_func_native(WASMModuleInstance *module_inst,
         return;
     }
 
-    if (func_import->call_conv_wasm_c_api) {
+    if (func_import->call_conv_wasm_c_api && c_api_func_import) {
```

Comment:
Same as above, seems no need to change here.

Reply:
done